go-controller/pkg/ovn/multipolicy_test.go

```diff
@@ -267,43 +269,68 @@ var _ = ginkgo.Describe("OVN MultiNetworkPolicy Operations", func() {
 			Name: nodeName,
 			UUID: nodeName + "_UUID",
 		}
-		logicalLayer2Switch := &nbdb.LogicalSwitch{
-			Name:        secondaryNetworkName + "_" + ovntypes.OVNLayer2Switch,
-			UUID:        secondaryNetworkName + "_" + ovntypes.OVNLayer2Switch + "_UUID",
-			ExternalIDs: map[string]string{ovntypes.NetworkExternalID: secondaryNetworkName},
-		}
 		initialData := getHairpinningACLsV4AndPortGroup()
 		initialData = append(initialData, logicalSwitch)
-		initialData = append(initialData, logicalLayer2Switch)
+		initialData = append(initialData, &nbdb.LogicalRouter{
+			Name: ovntypes.OVNClusterRouter,
+			UUID: ovntypes.OVNClusterRouter + "_UUID",
+		})
 		initialDB = libovsdb.TestSetup{
 			NBData: initialData,
 		}
+	})
+
+	ginkgo.AfterEach(func() {
+		fakeOvn.shutdown()
+		format.MaxLength = gomegaFormatMaxLength
+	})
+
+	setSecondaryNetworkData := func(topology, subnets string) {
```

Comment:
IMO we should either parametrize this func with the namespace/name or change it's name to reflect we're setting the network data of some specific NAD. 

By the time I saw it being called I was totally clueless as to where the "network data" was being set.